src/People/People.js

```diff
@@ -0,0 +1,79 @@
+import axios from 'axios'
+import React, { Component } from 'react'
+import "./people.css"
+
+export default class People extends Component {
+    constructor(){
+        super()
+        this.state={
+            searchperson: '',
+            currentperson: {},
+            namepers: '',
```

Comment:
Since the name of the person is stored inside of currentperson, this state variable isn't necessary. Try to refactor your code so that it works without using `this.state.namepers`